tests/functional/final-harvest.test.js

```diff
@@ -283,9 +283,8 @@ testDriver.test('final harvest sends resources', reliableResourcesHarvest.and(st
 
       return Promise.all([resourcesPromise, domPromise])
     })
-    .then((results) => {
-      t.equal(router.requestCounts.bamServer.resources, 2, 'received second resources harvest')
-      t.ok(results[0].request.body, 'received res harvest')
+    .then(([{ request: { body } }]) => {
+      t.ok(body, 'received res harvest')
```

Comment:
```suggestion
      t.ok(body, 'received second res harvest')
```
Makes test failures more clear on where problem is